File: 8_kyu/Multiplication_table_for_number.cs
using System;
using System.Linq;

public static class Kata
{
    // Generates multiplication table 1,2,..,10 times the number n.
    public static string MultiTable(int n) =>
        String.Join( "\n", Enumerable.Range(1,10).Select( i => $"{i} * {n} = {i*n}") );
}

File: 8_kyu/Multiplication_table_for_number.py
def multi_table(number: int) -> str :
    """
    Generates a multiplication table 1..10 times number.
    number -- base number for the multiplication table.
    return -- a string of 10 lines, each line a multiplication i*number=result as decimal numbers.
    """
    return "\n".join( f"{i} * {number} = {i*number}" for i in range(1,11) )
